datafusion/physical-expr/src/equivalence/properties.rs

```diff
@@ -2877,21 +3078,18 @@ mod tests {
                 &schema2,
             )
             .with_expected_sort_and_const_exprs(
-                // Union orderings:
-                // should be [a ASC]
+                // Union orderings: [a ASC]
                 //
-                // Where a, and a1 ath the same index for their corresponding
-                // schemas.
+                // Note that a, and a1 are at the same index for their
+                // corresponding schemas.
                 vec![vec!["a"]],
                 vec![],
             )
             .run()
     }
 
     #[test]
-    #[ignore]
-    // ignored due to https://github.com/apache/datafusion/issues/12446
```

Comment:
the tests now pass 🎉  -- I also renamed them to try and be more specific about what they are testing